package.json

```diff
@@ -72,17 +85,9 @@
       ]
     },
     "overrides": {
-      "extensions": [
-        "ts"
-      ],
       "extends": [
         "xo-typescript"
```

Comment:
This should be removed. It's bundled in XO.

Reply:
Yo, that fixes the problem.